src/amp/components/Epic.tsx

```diff
@@ -192,14 +315,55 @@ const buildUrl = (
 	)}`;
 };
 
+const getReminderDate = (date: Date = new Date()): Date => {
+	const monthsAhead = date.getDate() < 20 ? 1 : 2;
+	const reminderDate = new Date();
+	reminderDate.setMonth(date.getMonth() + monthsAhead);
+
+	return reminderDate;
+};
+
+const arrowSvg = (className: string) => (
+	<svg
+		className={className}
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 20 17.89"
+		preserveAspectRatio="xMinYMid"
+		aria-hidden="true"
+		focusable="false"
+	>
+		<path d="M20 9.35l-9.08 8.54-.86-.81 6.54-7.31H0V8.12h16.6L10.06.81l.86-.81L20 8.51v.84z" />
+	</svg>
+);
+
 export const Epic: React.FC<{ webURL: string }> = ({ webURL }) => {
-	const epicUrl =
-		process.env.GU_STAGE === 'PROD'
-			? 'https://contributions.guardianapis.com/amp/epic?ampVariantAssignments=VARIANTS'
-			: 'https://contributions.code.dev-guardianapis.com/amp/epic?ampVariantAssignments=VARIANTS';
+	const reminderDate = getReminderDate();
+	const reminderMonth = reminderDate.toLocaleString('default', {
+		month: 'long',
+	});
+	const reminderYear = reminderDate.getFullYear();
+	const epicStateJson = {
+		hideButtons: false,
+		hideReminderWrapper: true,
+		hideSuccessMessage: true,
+		hideFailureMessage: true,
+		hideReminderCta: false,
+		hideReminderForm: false,
+		headerText: `Remind me in ${reminderMonth} ${reminderYear}`,
+	};
+	const supportDotcomComponentsUrl = 'https://mjacobson.eu.ngrok.io';
```

Comment:
This URL doesn't look quite right?

Reply:
Haha, yeah, that's my testing URL. Some of the AMP things don't work without https. Don't worry, I'll remove it.